### PR TITLE
New design prototype

### PR DESCRIPTION
**Pull request by GabCh**
Macou want to try the old background with a new sphere, what do you think? the colors? have a design idea? animation we could do?

https://gabch.github.io/nodes-website/

### CODE REVIEW

File: src/sketches/ecophon.ts [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    }
+
+    private displayBackground = (): void => {
+        this.p5.noStroke()
+        this.p5.smooth()
+
+        for (let i = 0; i < this.PARTICLES_NUMBER; i++) {
+            const radius = this.p5.map(i, 0, this.PARTICLES_NUMBER, 1, 2)
+
+            this.p5.stroke('#db7d2f')
```
Comment (Gab05) [inline src/sketches/ecophon.ts]: Maybe store colors as constants in a separate file (enum or something) with readable names!

File: src/sketches/ecophon.ts [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+            this.particlesA[i].move()
+            this.particlesA[i].display(radius)
+            this.particlesA[i].checkEdge()
+
+            this.p5.stroke('#f9d500')
```
Comment (Gab05) [inline src/sketches/ecophon.ts]: Find a way to reduce code duplication

File: tslint.json [hunk 3]
```diff
@@ -38,7 +38,8 @@
         "member-access": false,
         "interface-name": false,
         "no-console": false,
-        "curly": false
+        "curly": false,
+        "no-bitwise": false
```
Comment (Gab05) [inline tslint.json]: Good idea, rules should be useful not bothering 👍

**Comment by GabCh** [comment]
Yes, great work for the new side menu!

**Pull request merged**